.github/workflows/deploy_and_Integration_test.yml

```diff
@@ -25,12 +25,15 @@ jobs:
           
       - name: Login to Cloud Foundry
         run: |
-          wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
-          echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
-          sudo apt-get update
-          sudo apt-get install cf8-cli
+          sudo apt-key del 172B5989FCD21EF8
```

Comment:
Is this a one-time command or we need to run it everytime?

Reply:
We have to run this every time as it uses new host every time.